app/views/internal/articles/_individual_article.html.erb

```diff
@@ -44,14 +44,16 @@
 		<% article.decorate.cached_tag_list_array.each do |tag| %>
 			<a href='/t/<%= tag %>'>#<%= tag %></a>
 		<% end %>
-		<a href="/internal/users/<%= article.user_id %>">@<%= article.user&.username %></a> ❤️ <%= article.positive_reactions_count %> | 💬 <%= article.comments_count %> | ID: <a href="/internal/articles/<%= article.id %>"><%= article.id %></a>
+		<a href="/internal/users/<%= article.user_id %>">@<%= article.user&.username %></a> ❤️ <%= article.positive_reactions_count %> | 💬 <%= article.comments_count %>
 		<br/><br/>
 		<div class="inner-row">
 			<form action="/internal/articles/<%= article.id %>" accept-charset="UTF-8" method="post">
 				<input name="utf8" type="hidden" value="✓">
 				<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>" />
 				<input type="hidden" name="_method" value="patch" />
 				Featured Number: <input name="article[featured_number]" value="<%= article.featured_number %>" id="featured_number_<%= article.id %>" style="max-width:140px"><button onclick="timeNow(<%= article.id %>);return false;">☝️</button><button onclick="sink(<%= article.id %>);return false;">👇 SPAM!</button>
+				<a href="/internal/articles/<%= article.id %>/edit">
```

Comment:
Missing closing `</a>` here causing everything thereafter to be part of the link

![](https://cl.ly/139a5ad61162/Image%202019-01-23%20at%209.15.06%20AM.png)

Was this link even supposed to be there?

Reply:
haha whoops! i had caught that but forgot to push again.